domain-service/src/main/kotlin/pet/project/app/mapper/UserControllerMapper.kt

```diff
@@ -0,0 +1,49 @@
+package pet.project.app.mapper
+
+import pet.project.app.model.domain.DomainUser
+import pet.project.internal.commonmodels.user.user.User
+import pet.project.internal.input.reqreply.user.add_book_to_wish_list.AddBookToUsersWishListResponse
+import pet.project.internal.input.reqreply.user.create.CreateUserResponse
+import pet.project.internal.input.reqreply.user.delete.DeleteUserByIdResponse
+import pet.project.internal.input.reqreply.user.find.FindUserByIdResponse
+import pet.project.internal.input.reqreply.user.update.UpdateUserResponse
+
+object UserControllerMapper {
+
+    fun DomainUser.toCreateUserResponse(): CreateUserResponse {
+        return CreateUserResponse.newBuilder().also {
+            it.successBuilder.user = this.toProto()
+        }.build()
+    }
+
+    fun DomainUser.toProto(): User {
+        return User.newBuilder().also {
+            it.setId(id)
+            it.setEmail(email)
+            it.setLogin(login)
+            it.addAllBookWishList(bookWishList)
```

Comment:
Make code more kotlinish
```suggestion
            it.id = id
            it.email - email
            it.login = login
            it.addAllBookWishList(bookWishList)
```